custom/abt/reports/late_pmt_2020.py

```diff
@@ -106,11 +106,7 @@ def _to_report_format(date, location, error_msg):
                 error_msg
             ]
 
-        include_missing_pmt_data = self.report_config['submission_status'] != 'incorrect_pmt_data'
-        # include_incorrect_pmt_data is no longer applicable, because an
-        # invalid SMS will not result in a pmt_data case being created/updated
-        include_incorrect_pmt_data = self.report_config['submission_status'] != 'missing_pmt_data'
-        error_msg = _('Incorrect or no PMT data submitted')
+        show_all = self.report_config['submission_status'] != 'missing_pmt_data'
```

Comment:
Is this a new option?  I thought the report never showed "good" PMTs

Reply:
The filter's default value is "All". If I've misunderstood that (and given that the name of the report is "**Late** PMT" I guess I have) I should just drop the filter.

Reply:
I think "All" means "all missing or incorrect".  In the existing behavior, "good" PMTs are omitted from results.
https://github.com/dimagi/commcare-hq/blob/3a62c51a6771031bedccb17c04ef36392c827efd/custom/abt/reports/late_pmt.py#L217-L217